### PR TITLE
Add 1.11 release notes

### PR DESCRIPTION
**Pull request by angerson**
This change adds the release notes for 1.11, which were mostly shepherded by a new internal automation tool. Cool!

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
+## Breaking Changes
+
+* Keras:
+  * The default values for tf.keras `RandomUniform`, `RandomNormal`, and `TruncatedNormal` initializers have been changed to match those in external Keras.
+* `tf.contrib`:
+  * Breaking change: `model.get_config()` on a Sequential model now returns a config dictionary (consistent with other Model instances) instead of a list of configs for the underlying layers.
```
Comment (gunan) [inline RELEASE.md]: We should reference the full location of this. Is this tf.contrib.model?

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+
+## Major Features and Improvements
+
+* Cloud TPU:
+  * Experimental tf.data integration for Keras on Cloud TPUs.
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
+## Breaking Changes
```
Comment (aaroey) [inline RELEASE.md]: I think we'll need to add cudnn 7.2 changes, maybe something like:
```
Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
```
Also @tfboyd .

File: RELEASE.md (hunk 1 above)
Comment (angersson) [reply RELEASE.md]: Huh, looks like this was in Keras but got put under tf.contrib. Updated in the upcoming push.

File: RELEASE.md (hunk 2 above)
Comment (angersson) [reply RELEASE.md]: Added in the upcoming push, thanks!

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Cloud TPU:
+  * Experimental tf.data integration for Keras on Cloud TPUs.
+* Add C functions for querying kernels
+* Add C++ functions for querying kernels
+* Add Python functions for querying kernels
+
```
Comment (anj-s) [inline RELEASE.md]: Can we add the following items to the Release notes? Thanks!
* DistributionStrategy
    * Add multi-GPU DistributionStrategy support in tf.keras. Users can now use `fit`, `evaluate` and `predict` to 
      distribute their model on multiple GPUs.
    * Add multi-worker  DistributionStrategy and standalone client support in Estimator. See  [README] 
      (https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute) for more details.

Comment (angersson) [reply RELEASE.md]: Done, thanks!

File: RELEASE.md [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+  * GoogleZoneProvider class added to detect  which Google Cloud Engine zone tensorflow is running in.
+  * It is now safe to call any of the C API's TF_Delete\* functions on nullptr
+  * Log some errors on Android to logcat
+  * Match FakeQuant numerics in TFLite to improve accuracy of TFLite quantized inference models.
+  * Optional bucket location check for the GCS Filesystem.
+  * Performance enhancements for StringSplitOp & StringSplitV2Op.
+  * Performance improvements for regex replace operations.
+  * TFRecordWriter now raises an error if .write() fails.
+  * TPU: More helpful error messages in TPUClusterResolvers.
+  * The legacy_init_op argument to SavedModelBuilder methods for adding MetaGraphs has been deprecated. Please use the equivalent main_op argument instead. As part of this, we now explicitly check for a single main_op or legacy_init_op at the time of SavedModel building, whereas the check on main_op was previously only done at load time. SKIP_ATC=Ignoring the Tap cnn_e2e_test error as per the recommendations of chendouble@, who says they will regenerate the SavedModel protos in question after the fact.
```
Comment (martinwicke) [inline RELEASE.md]: I think some bad language crept in here, probably a bad description parse

File: RELEASE.md [hunk 5]
```diff
@@ -1,3 +1,76 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
+* Cloud TPU:
```
Comment (rajatmonga) [inline RELEASE.md]: Change "Cloud TPU" here and in next line to "Google Cloud TPU".

File: RELEASE.md [hunk 6]
```diff
@@ -1,3 +1,76 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
```
Comment (rajatmonga) [inline RELEASE.md]: Add "Nvidia GPU: " to signify that this change is specific to Nvidia GPUs.

**Review by rajatmonga — APPROVED** [review]

File: RELEASE.md (hunk 6 above)
Comment (aaroey) [inline RELEASE.md]: Sorry I missed the TensorRT change, so maybe:
`...built against cuDNN 7.2 and TensorRT 4`

File: RELEASE.md (hunk 4 above)
Comment (angersson) [reply RELEASE.md]: I found the internal change to double-check -- the SKIP_ATC is supposed to be separate. Thanks!

File: RELEASE.md (hunk 5 above)
Comment (angersson) [reply RELEASE.md]: Done, thanks!

File: RELEASE.md (hunk 6 above)
Comment (angersson) [reply RELEASE.md]: Done, thanks!

Comment (angersson) [reply RELEASE.md]: Done, thanks!

File: RELEASE.md [hunk 7]
```diff
@@ -1,3 +1,77 @@
+# Release 1.11.0
+
+## Major Features and Improvements
+
+* Nvidia GPU:
+  * Prebuilt binaries are now (as of TensorFlow 1.11) built against cuDNN 7.2 and TensorRT 4. See updated install guides: [Installing TensorFlow on Ubuntu](https://www.tensorflow.org/install/install_linux#tensorflow_gpu_support)
+* Google Cloud TPU:
+  * Experimental tf.data integration for Keras on Google Cloud TPUs.
```
Comment (asimshankar) [inline RELEASE.md]: Also:
  * Experimental / preview support for eager execution on Google Cloud TPUs

Comment (angersson) [reply RELEASE.md]: Done, thanks!

**Pull request merged**